scripts/amm/stablepool/README.md

```diff
@@ -0,0 +1,16 @@
+## Stablepool deployment
+
+1. Export wallet to a JSON format and save it as `scripts/amm/stablepool/deployerWallet.json`.
+2. Create `scripts/secrets.json` and as shown in `scripts/amm/stablepool/secrets.example.json`.
```

Comment:
If you chose to supply `seed` then the only additional var is RPC_URL which can be set via URL as well.